services/workflow.go

```diff
@@ -21,10 +20,12 @@ var caseTypeMapping = map[string]int{
 var Workflows []Workflow
 
 type Workflow struct {
-	TopicName    string
-	Offset       int
-	FunctionName string
-	Enabled      bool
+	Id           uint32 `json:"id"`
```

Comment:
Let's just make this an int for now. It will simplify having to parse ints -> uint32 later in the code. I get that it's more efficient but for now we don't need this optimized.